R/linux.R

```diff
@@ -140,7 +140,20 @@ psl__decode_address <- function(addr, family) {
   }
 }
 
-ps_cpu_count_physical_linux <- function() {
+psl__cpu_count_from_lscpu <- function() {
+  lines <- system("lscpu -p=core", intern = TRUE)
+  cores = list()
+
+  for (l in lines) {
+    if (!str_starts_with(l, "#")) {
+      cores[str_strip(l)] <- 1
+    }
+  }
```

Comment:
Not sure about the style here; this loop could maybe also be written as `cores <- unique(lines[!str_starts_with(lines, "#")])`

Reply:
Yeah, I'll change it to that.